### PR TITLE
roachtest: more ground work to move tests into `tests`

### PR DESCRIPTION
**Pull request by tbg**
See individual commits.

### CODE REVIEW

File: pkg/cmd/roachtest/option/options.go [hunk 1]
```diff
@@ -39,3 +39,19 @@ var StartArgsDontEncrypt = StartArgs("--encrypt=false")
...
+// WorkerAction informs a cluster operation that the callee is a "worker" rather
+// than the test's main goroutine.
+type WorkerAction struct{}
+
+var _ Option = WorkerAction{}
+
+// Option implements Option.
+func (o WorkerAction) Option() {}
+
+// withWorkerAction creates the workerAction option, informing a cluster
```
Comment (erikgrinaker) [inline pkg/cmd/roachtest/option/options.go:52]:
```suggestion
// WithWorkerAction creates a WorkerAction option, informing a cluster
```

File: pkg/cmd/roachtest/tests/util.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
+	"github.com/cockroachdb/errors"
+)
+
+// WaitFor3XReplication waits until all ranges in the system are on at least
+// three voters.
+//
+// TODO(nvanbenschoten): this function should take a context and be responsive
+// to context cancellation.
+func WaitFor3XReplication(t test.Test, db *sql.DB) {
```
Comment (erikgrinaker) [inline pkg/cmd/roachtest/tests/util.go:29]: Like the name change. Should eventually have a variant that detects the expected number of replicas.

**Review by erikgrinaker — APPROVED** [review]
LGTM. The two last commits could arguably be merged, but 🤷‍♀️.

**Comment by tbg** [comment]
TFTR! Incorporated your suggestion. "Unfortunately", about the time you were reviewing, I pushed another batch of commits. Mind taking a look at them too and signing off again? "roachtest: fix (another) stray newMonitor" is the first new commit I think.

**Review by erikgrinaker — APPROVED** [review]

**Comment by tbg** [comment]
bors r=erikgrinaker

**Pull request merged**